internal/plugins/ansible/v1/scaffolds/init.go

```diff
@@ -102,7 +108,11 @@ func (s *initScaffolder) scaffold() error {
 		&kdefault.Kustomize{},
 		&kdefault.AuthProxyPatch{},
 
-		&templates.Makefile{},
+		&templates.Makefile{
+			KustomizeVersion:       KustomizeVersion,
+			Image:                  imageName,
+			AnsibleOperatorVersion: AnsibleOperatorVersion,
```

Comment:
It is missing pass the AnsibleOperatorVersion to the docker file as well.

Reply:
Good point.